gulpfile.js

```diff
@@ -0,0 +1,162 @@
+const autoprefixer = require('gulp-autoprefixer');
+const babelify = require('babelify');
+const browserify = require('browserify');
+const buffer = require('vinyl-buffer');
+const cache = require('gulp-cache');
+const del = require('del');
+const eslint = require('gulp-eslint');
+const globby = require('globby');
+const gulp = require('gulp');
+const gulpif = require('gulp-if');
+const gutil = require('gulp-util');
+const imagemin = require('gulp-imagemin');
+const inject = require('gulp-inject');
+const minifycss = require('gulp-minify-css');
+const rename = require('gulp-rename');
+const runSequence = require('run-sequence');
+const sass = require('gulp-sass');
+const source = require('vinyl-source-stream');
+const sourcemaps = require('gulp-sourcemaps');
+const through = require('through2');
+const uglify = require('gulp-uglify');
+const connect = require('gulp-connect');
+
+const IS_DEBUG = process.env.NODE_ENV !== 'production';
+
+const SRC_PATH = './src/';
+const DEST_PATH = './dist/';
+
+// Lint the gulpfile
+gulp.task('selfie', function selfieTask() {
+  return gulp.src('gulpfile.js')
+    .pipe(eslint())
+    .pipe(eslint.format());
+});
+
+gulp.task('lint', function lintTask() {
+  return gulp.src(['*.js', SRC_PATH + 'app/**/*.js'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failOnError());
+});
+
+gulp.task('clean', function cleanTask(done) {
+  del([
+    DEST_PATH
+  ], done);
+});
+
+gulp.task('npm:tabzilla:img', function npmTabzillaImgTask() {
+  return gulp.src('./node_modules/mozilla-tabzilla/media/**')
+    .pipe(gulp.dest(DEST_PATH + 'vendor/mozilla-tabzilla/media'));
+});
+
+// Copy the tabzilla assets into the src dir for inclusion in minimization
+gulp.task('npm:tabzilla:css', function npmTabzillaCssTask() {
+  return gulp.src('./node_modules/mozilla-tabzilla/css/tabzilla.css')
+    .pipe(rename('mozilla-tabzilla/css/tabzilla.scss'))
+    .pipe(gulp.dest(SRC_PATH + 'styles/vendor'));
+});
+
+// Copy the normalize assets into the src dir for inclusion in minimization
+gulp.task('npm:normalize', function npmNormalizeTask() {
+  return gulp.src('./node_modules/normalize.css/normalize.css')
+    .pipe(rename('normalize.css/normalize.scss'))
```

Comment:
I think this is causing a mid-air collision with some inject flow.
When I try and run this branch using `npm start` I see that we're trying to load 'normalize.css' twice, but one of the requests returns a 303 ("See Other").

1. <http://localhost:9988/styles/vendor/normalize.css> (which looks like a file, but is actually a directory listing which gives me a **303 GET** response).
2. <http://localhost:9988/styles/vendor/normalize.css/normalize.css> (which is the actual CSS file and gives me a **200 GET** response).

We may want to try renaming this to "normalize/normalize.scss" so it doesn't match the **html-pages** task's CSS glob. Or maybe it's all moot and @johngruen will just be `@import`ing the vendor files anyways.